Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -42,17 +42,17 @@ ProjectPamphletViewModelOutputs {
 
   // swiftlint:disable:next function_body_length
   public init() {
-    let configData = Signal.combineLatest(
+    let configDataAndShouldPrefix = Signal.combineLatest(
       self.configDataProperty.signal.skipNil(),
-      Signal.merge(self.viewDidLoadProperty.signal,
-                   self.viewDidAppearAnimated.signal.filter(isTrue).ignoreValues()
+      Signal.merge(self.viewDidLoadProperty.signal.mapConst(true),
+                   self.viewDidAppearAnimated.signal.filter(isTrue).mapConst(false)
```

Comment:
i still think we should not emit `configData` multiple times cause the input is only called a single time. we could move the `Signal.merge` to a `takePairWhen` in the `freshProjectAndLiveStreamsAndRefTag` signal. also helps localize the `shouldPrefix` logic to near where it's needed